### DISCUSSION TITLE
USA CDJ

### DISCUSSION THREAD

**Discussion by gabadt** [LorenzoBalducci96/custom-cdj#8] [Q&A]
hi, thanks for the sharing !
I'm looking to achieve this project and i was wondering if i used CDJ from USA (110v) would change something or i could follow your pdf and do the exact same.
If you know, thanks !

See u

**Comment by LorenzoBalducci96** [comment]
You have to completely remove the power supply section of the cdj and the "button shield" of the CDJ is completely "passive".
So I would say yes, there is no difference.
